peakdet/__init__.py

```diff
@@ -12,7 +12,11 @@
                                 interpolate_physio, peakfind_physio,
                                 plot_physio, reject_peaks)
 from peakdet.physio import (Physio)
+from loguru import logger
 
 from ._version import get_versions
 __version__ = get_versions()['version']
-del get_versions
\ No newline at end of file
+del get_versions
+
+# TODO: Loguru does not detect the module's name
```

Comment:
Is this something you still need to work on?

Reply:
Yes, so it's not something urgent imo or breaking. Essentially because you need to disable by default the logs when the module is used as a library, you call the disable function as in [here](https://loguru.readthedocs.io/en/stable/resources/recipes.html#configuring-loguru-to-be-used-by-a-library-or-an-application). However for some cases the logger cannot detect the name and there is [this workaround](https://loguru.readthedocs.io/en/stable/resources/recipes.html#circumventing-modules-whose-name-value-is-absent). It did not really work and the workaround I found was the line after the TODO.

It should be a straightforward fix after some troubleshooting, but since the current one is functional I would say that we open an issue and I can work on it in the near future

Reply:
This is now fixed @m-miedema